src/Generator/MethodGenerator.php

```diff
@@ -142,6 +144,9 @@ public static function fromArray(array $array)
                     break;
                 case 'final':
                     $method->setFinal($value);
+                case 'interface':
```

Comment:
It can't be done because clearly there is a bug here. missing break statement and later duplicate.

Please don't rebase until the PR review ends or maintainer request it.